src/govuk/components/back-link/_index.scss

```diff
@@ -1,16 +1,19 @@
 @include govuk-exports("govuk/component/back-link") {
+  // Component font-size on the Frontend (used for calculations)
+  $font-scale-size: 16;
+  $font-size: $font-scale-size * 1px;
 
   // Size of chevron (excluding border)
-  $chevron-size: 7px;
+  $chevron-size: govuk-em(7px, $font-size);
 
   // Size of chevron border
-  $chevron-border-width: 1px;
+  $chevron-border-width: govuk-em(1px, $font-size);
```

Comment:
Are we back to having the thinner-than-1px border now?

Reply:
Unfortunately yes. The method using `max` didn't work out for the Breadcrumbs component, due to some of the ways it tries to use the value of 

As the chevron is intended to be visually identical between the Breadcrumbs and the Back Link, and how often these elements appear next to one another, being consistently too narrow seemed better than being inconsistent.

Granted, I may have just thought of a way around this, so I'm gonna give that a shot now 🤔